src/connectors/voice-search/__tests__/connectVoiceSearch-test.js

```diff
@@ -77,12 +78,19 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/voice-searc
       );
     });
 
-    it('calls unmount on dispose', () => {
+    it('calls unmount on disposal', () => {
```

Comment:
reads pretty weird, I think we can keep `dispose` (maybe even in backticks to signify the fact that it's a function)

Reply:
haha okay. I've reverted it back to disposal with backticks 👍 
https://github.com/algolia/instantsearch.js/pull/3779/commits/8aaab1c492e7d76e7af4b603a4bb33f5a2b74ab3